### DISCUSSION TITLE
Conda Package 1.6.1

### DISCUSSION THREAD

**Discussion by jc211** [pthom/imgui_bundle#299] [General]
Any chance of pushing a 1.6.1 package to conda. 1.6.0 is bugged and can't be used for manipulate :(

**Comment by pthom** [comment]
Hello, 

I am not in charge (yet, anyhow), of the conda package. I was hugely assisted in it by @hmaarrfk who was in charge the conda package 1.6.0.

In the meantime my advice is to use pip to install v1.6.1

**Reply by hmaarrfk** [reply]
thanks for the ping. i wasn't aware of bugs so i was kinda "lazy" and wanted to see if the bot would issue a PR updating things for me before diving into it.

I can look at it this weekend.

@jc211 if you have some git-fu you can try to make a PR to conda-forge's feedstock and I can review it.

The biggest challenge will be to rebase the patches.

**Comment by pthom** [comment]
Hi @hmaarrfk thanks for your answer ! 

I will not be available to help you this weekend, as I am quite sick since a few days and I am a bit stuck to my bed.

**Reply by hmaarrfk** [reply]
hopefully you recover quickly to be able to enjoy the new year!

**Comment by hmaarrfk** [comment]
I just released 1.6.2 let us know how well it works! pthom helped upstream many of the patches i had!
